pom.xml

```diff
@@ -194,10 +196,7 @@
         <springframework.test.version>3.2.18.RELEASE</springframework.test.version>
         <springframework.version>3.2.18.RELEASE</springframework.version>
         <storm.version>1.0.2</storm.version>
-        <sun-jersey-bundle.version>1.19</sun-jersey-bundle.version>
-        <sun.jersey.bundle.version>1.4</sun.jersey.bundle.version>
-        <sun.jersey.core.version>1.4</sun.jersey.core.version>
-        <sun.jersey.spring.version>1.4</sun.jersey.spring.version>
+        <tez.version>0.5.2</tez.version>
```

Comment:
do we really need TEZ dependency? I remember myself removing it from our branch. Let's verify its necessity.